features/preferences/impl/src/main/kotlin/io/element/android/features/preferences/impl/labs/LabsEvents.kt

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright 2025 New Vector Ltd.
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only OR LicenseRef-Element-Commercial
+ * Please see LICENSE files in the repository root for full details.
+ */
+
+package io.element.android.features.preferences.impl.labs
+
+sealed interface LabsEvents {
+    data class ToggleFeature(val feature: String) : LabsEvents
```

Comment:
I will not use the `Feature.key` for the parameter to avoid mistake if a developer use `Feature.title`. I would change the API to 

```suggestion
    data class ToggleFeature(val feature: FeatureUiModel) : LabsEvents
```